### PR TITLE
fix: [PIE-4628]: Remove validation of missing runtime default values

### PR DESCRIPTION
**Pull request by Fernando-Dourado**
## Describe your changes
- Remove validation of missing runtime default values

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/39079)

### CODE REVIEW

**Review by archit-harness — DISMISSED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39079)* status: 3 file(s) to review. 1 of 2 approvals obtained.

File: pipeline-service/service/src/main/java/io/harness/pms/ngpipeline/inputset/helpers/InputSetErrorsHelper.java [hunk 1]
```diff
@@ -189,12 +190,6 @@ Map<FQN, String> getInvalidFQNsInInputSetFromTemplateConfig(
 
         inputSetFQNs.remove(key);
       } else {
-        if (!key.isType() || !key.isIdentifierOrVariableName()) {
```
Comment (namanvermah) [inline pipeline-service/service/src/main/java/io/harness/pms/ngpipeline/inputset/helpers/InputSetErrorsHelper.java:192]: Why are we removing this?

Comment (Fernando-Dourado) [reply pipeline-service/service/src/main/java/io/harness/pms/ngpipeline/inputset/helpers/InputSetErrorsHelper.java:192]: It's the rollback from the previous changes, this is the original method content.

**Review by namanvermah — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39079)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Comment by Fernando-Dourado** [comment]
trigger pmd

**Review by archit-harness — APPROVED** [review]
> Reviewed 1 of 2 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39079)* status: 1 file(s) to review. 1 of 2 approvals obtained.

**Review by namanvermah — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39079)* status: Some of the checks are too old.

**Comment by Fernando-Dourado** [comment]
In develop branch this issue is not reproducible anymore.

**Pull request closed without merging**